File: src/main/java/com/iflat/xr/entity/QuotaManhour.java
package com.iflat.xr.entity;

import java.util.Date;

/**
 * Created by tyriv on 2016/8/28.
 */
public class QuotaManhour {
    private String projNo;
    private String projName;
    private String deptCode;
    private String dept;
    private String teamCode;
    private String team;
    private double quota;
    private String creatorAcc;
    private String creatorName;
    private Date createTime;
    private Date settlementTime;

    private Date fromDate;
    private Date toDate;

    public Date getFromDate() {
        return fromDate;
    }

    public void setFromDate(Date fromDate) {
        this.fromDate = fromDate;
    }

    public Date getToDate() {
        return toDate;
    }

    public void setToDate(Date toDate) {
        this.toDate = toDate;
    }

    public String getProjNo() {
        return projNo;
    }

    public void setProjNo(String projNo) {
        this.projNo = projNo;
    }

    public String getProjName() {
        return projName;
    }

    public void setProjName(String projName) {
        this.projName = projName;
    }

    public String getDeptCode() {
        return deptCode;
    }

    public void setDeptCode(String deptCode) {
        this.deptCode = deptCode;
    }

    public String getDept() {
        return dept;
    }

    public void setDept(String dept) {
        this.dept = dept;
    }

    public String getTeamCode() {
        return teamCode;
    }

    public void setTeamCode(String teamCode) {
        this.teamCode = teamCode;
    }

    public String getTeam() {
        return team;
    }

    public void setTeam(String team) {
        this.team = team;
    }

    public double getQuota() {
        return quota;
    }

    public void setQuota(double quota) {
        this.quota = quota;
    }

    public String getCreatorAcc() {
        return creatorAcc;
    }

    public void setCreatorAcc(String creatorAcc) {
        this.creatorAcc = creatorAcc;
    }

    public String getCreatorName() {
        return creatorName;
    }

    public void setCreatorName(String creatorName) {
        this.creatorName = creatorName;
    }

    public Date getCreateTime() {
        return createTime;
    }

    public void setCreateTime(Date createTime) {
        this.createTime = createTime;
    }

    public Date getSettlementTime() {
        return settlementTime;
    }

    public void setSettlementTime(Date settlementTime) {
        this.settlementTime = settlementTime;
    }
}

File: web/app/model/xr/QuotaManhour.js
Ext.define('iFlat.model.xr.QuotaManhour', {
    extend: 'Ext.data.Model',
    fields: [
        {name: 'quotaManhour.projNo', mapping: 'projNo', type: 'string'},
        {name: 'quotaManhour.projName', mapping: 'projName', type: 'string'},
        {name: 'quotaManhour.deptCode', mapping: 'deptCode', type: 'string'},
        {name: 'quotaManhour.dept', mapping: 'dept', type: 'string'},
        {name: 'quotaManhour.teamCode', mapping: 'teamCode', type: 'string'},
        {name: 'quotaManhour.team', mapping: 'team', type: 'string'},
        {name: 'quotaManhour.quota', mapping: 'quota', type: 'number'},
        {name: 'quotaManhour.creatorAcc', mapping: 'creatorAcc', type: 'string'},
        {name: 'quotaManhour.creatorName', mapping: 'creatorName', type: 'string'},
        {name: 'quotaManhour.createTime', mapping: 'createTime', type: 'date'},
        {name: 'quotaManhour.settlementTime', mapping: 'settlementTime', type: 'date'},
        {name: 'quotaManhour.fromDate', mapping: 'fromDate', type: 'date'},
        {name: 'quotaManhour.toDate', mapping: 'toDate', type: 'date'},
    ]
});